projects/go-style-guide/src/app/features/ui-kit/components/accordion-docs/components/accordion-overview/accordion-overview.component.html

```diff
@@ -1,106 +1,116 @@
+        <p class="go-body-copy">
+          This will enable each panel to have an icon shown to the left of the heading.
+        </p>
+        <caption>
+          Note: even if the <code class="code-block--inline">icon</code> binding is passed to an
+          <code class="code-block--inline">accordion-panel</code>, it will not be displayed unless
+          <code class="code-block--inline">showIcons</code> is set to <code class="code-block--inline">true</code>.
+          This is to encourage a consistent UI. If one <code class="code-block--inline">accordion-panel</code>
+          has an icon, they all should.
+        </caption>
+      </div>
 
-      <!-- slim -->
-      <h4 class="go-heading-4">slim</h4>
-      <p class="go-body-copy">
-        <code class="code-block--inline">slim</code>
-        will reduce the height of the accordion panel, making it slimmer vertically.
-      </p>
+      <div class="go-column go-column--100">
+        <!-- slim -->
+        <h4 class="go-heading-4">slim</h4>
+        <p class="go-body-copy">
+          <code class="code-block--inline">slim</code>
+          will reduce the height of the accordion panel, making it slimmer vertically.
+        </p>
+      </div>
 
-      <!-- theme -->
-      <h4 class="go-heading-4">theme</h4>
-      <code [highlight]="themeOptions"></code>
-      <p class="go-body-copy">
-        This will set the appearance of the <code class="code-block--inline">accordion</code>. <code class="code-block--inline">light</code>
-        is the default <code class="code-block--inline">theme</code> and should be used in most places. The only time the
-        <code class="code-block--inline">dark</code> theme should be used is when the accordion
-        lives inside a UI element that has a dark themed background by default. (ex: off-canvas)
-      </p>
+      <div class="go-column go-column--100 go-column--no-padding">
+        <!-- theme -->
+        <h4 class="go-heading-4">theme</h4>
+        <code [highlight]="themeOptions"></code>
+        <p class="go-body-copy">
+          This will set the appearance of the <code class="code-block--inline">accordion</code>. <code class="code-block--inline">light</code>
+          is the default <code class="code-block--inline">theme</code> and should be used in most places. The only time the
+          <code class="code-block--inline">dark</code> theme should be used is when the accordion
+          lives inside a UI element that has a dark themed background by default. (ex: off-canvas)
+        </p>
+      </div>
     </div>
   </go-card>
 
   <go-card class="go-column go-column--100">
     <header go-card-header>
-      <h3 class="go-heading-3">Default</h3>
+      <h2 class="go-heading-2">Default</h2>
     </header>
     <div go-card-content>
       <div class="go-container">
         <div class="go-column go-column--50">
-          <h4 class="go-heading-4">Code</h4>
+          <h3 class="go-heading-3">Code</h3>
```

Comment:
In forms we're using an h4 with a class of `go-heading--underlined` for the code and view headers. Should we make these consistent?

Reply:
Sure, I fixed it.